authors/apps/profiles/views.py

```diff
@@ -61,3 +61,68 @@ class UserProfileListView(generics.ListAPIView):
     @classmethod
     def get_queryset(self):
         return Profile.objects.all()
+
+
+class FollowersAPIView(generics.RetrieveAPIView):
+    """A class for actors who request.user is following."""
+    permission_classes = (IsAuthenticated,)
+    serializer_class = FollowerSerializer
+    lookup_field = 'username'
```

Comment:
@ManuelDominic, use a renderer class to return a response.
it should be in the form
```
{
"followers":[
{<user profile here >},
{<another user profile here >},
...
]
            
}
```